CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Changelog
 
+## 15.1.9 (03/19/24)
+
+* Improved performance when listing nodes with tsh or tctl. [#39567](https://github.com/gravitational/teleport/pull/39567)
+* Fixed a bug where AWS S3 bucket fields were not required when creating or editing AWS OIDC integrations in the web UI. [#39510](https://github.com/gravitational/teleport/pull/39510)
```

Comment:
This is a new feature, not a bug.

Reply:
This was listed on https://github.com/gravitational/teleport/pull/39285 as a bug fix. What should the line read as? The original changelog value from the PR is `Require AWS S3 bucket fields when creating/editing AWS OIDC integration in the web UI`.

Reply:
The original changelog line is ok, I would just keep it as-is.

Reply:
@r0mant done - please stamp when you get a chance